src/extension/components/__tests__/finder.ts

```diff
@@ -6,7 +6,7 @@ import { createFinder } from "../finder";
 import { DeoptEntry } from "../../../third-party-derived/deoptigate/deoptEntry";
 import { Entry } from "../../model/entry";
 
-it("finder", () => {
+it.skip("finder", () => {

```

Comment:
Is this skip intentional?

Reply:
Yes, this test is somewhat dated and I need to review it as part of adding more tests to the project long term.